### PR TITLE
gogitops updates

### PR DESCRIPTION
**Pull request by Beaujr**

### CODE REVIEW

**Review by Beaujr — COMMENTED** [review]
outdated

File: main.go [hunk 1]
```diff
@@ -30,6 +30,6 @@ func main() {
 	} else {
 		myRouter := mux.NewRouter().StrictSlash(true)
 		myRouter.Handle("/upload", xclient.HandleRequests(x))
-		http.ListenAndServe(":10000", myRouter)
+		err = http.ListenAndServe(":10000", myRouter)
```
Comment (gogitops) [inline main.go]: warning: ineffectual assignment to err (ineffassign)

File: main.go [hunk 2]
```diff
@@ -1,14 +1,16 @@
 package main
```
Comment (gogitops) [inline main.go]: warning: file is not gofmted with -s (gofmt)

File: main.go [hunk 3]
```diff
@@ -1,14 +1,16 @@
...
 	"fmt"
 	xclient "github.com/beaujr/go-xerox-upload/client"
 	"github.com/gorilla/mux"
 	"google.golang.org/appengine"
-	"net/http"
+	"log"
 	"os"
 )
 
+// This is a Simple Mispelling
```
Comment (gogitops) [inline main.go]: warning: "Mispelling" is a misspelling of "Misspelling" (misspell)

**Review by Beaujr — COMMENTED** [review]
outdated

File: main.go (hunk 2 above)
Comment (Beaujr) [inline main.go]: warning: file is not gofmted with -s (gofmt)

File: main.go (hunk 3 above)
Comment (Beaujr) [inline main.go]: warning: "Mispelling" is a misspelling of "Misspelling" (misspell)

**Review by Beaujr — COMMENTED** [review]
outdated

File: main.go (hunk 2 above)
Comment (Beaujr) [inline main.go]: warning: file is not gofmted with -s (gofmt)

File: main.go (hunk 3 above)
Comment (Beaujr) [inline main.go]: warning: "Mispelling" is a misspelling of "Misspelling" (misspell)

**Review by Beaujr — COMMENTED** [review]
outdated

File: main.go (hunk 2 above)
Comment (Beaujr) [inline main.go]: warning: file is not gofmted with -s (gofmt)

File: main.go (hunk 3 above)
Comment (Beaujr) [inline main.go]: warning: "Mispelling" is a misspelling of "Misspelling" (misspell)

**Review by Beaujr — COMMENTED** [review]
# GoGitOps Review

## Grade: A+ (92.6%)

### Files: 8

### Issues: 2

### gofmt: 100%

### go_vet: 75%

	client/drive.go
		Line 8: error: cannot find package "golang.org/x/oauth2/google" in any of: (vet)
	client/drive.go
		Line 9: error: cannot find package "google.golang.org/api/drive/v3" in any of: (vet)
	main.go
		Line 5: error: cannot find package "github.com/beaujr/go-xerox-upload/client" in any of: (vet)
	main.go
		Line 7: error: cannot find package "google.golang.org/appengine" in any of: (vet)
### gocyclo: 100%

### golint: 100%

### ineffassign: 100%

### license: 100%

### misspell: 100%

This report was generated using [GoGitOps](https://github.com/gogitops/gogitops-examples).

**Pull request merged**